atomsci/ddm/pipeline/chem_diversity.py

```diff
@@ -294,8 +294,7 @@ def upload_distmatrix_to_DS(
     dist_df = pd.DataFrame(dist_matrix)
     dist_df.index = compound_ids
     dist_df.columns = compound_ids
-    fnm = "distmatrix_nm"
-    filename = fn.replace("nm",feature_type)
-    dist_pkl = dist_df.to_pickle(filepath+filename)
```

Comment:
I think this should be:

```python
_dist_pkl = dist_df.to_pickle(filepath+filename)
```

The side-affect here is creating a file. [pickle](https://docs.python.org/3/library/pickle.html) is a library used to serialize Python objects so that you can de-serialize them at a later time